### PR TITLE
feat: support ctrl n/p on mac

### PR DESCRIPTION
**Pull request by wendellhu95**
Use <kbd>ctrl + n / p</kbd> to select list items. Closing https://github.com/ant-design/ant-design/issues/31641.

### CODE REVIEW

**Comment by wendellhu95** [comment]
@afc163 @zombieJ Could you please take a look at this?

**Comment by afc163** [comment]
覆盖率掉了

**Comment by wendellhu95** [comment]
> 覆盖率掉了

那我在测试环境里 mock 一下 platformUtil 这个文件吧。

**Comment by wendellhu95** [comment]
> 覆盖率掉了

Codecov 似乎得手动重跑一下

File: src/utils/platformUtil.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/* istanbul ignore file */
+export function isPlatformMac(): boolean {
+  return /mac\sos/i.test(navigator.appVersion);
```
Comment (afc163) [inline src/utils/platformUtil.ts:3]: 图片

File: src/OptionList.tsx [hunk 2]
```diff
@@ -183,16 +184,24 @@ const OptionList: React.RefForwardingComponent<
...
+        case KeyCode.N:
+        case KeyCode.P:
         case KeyCode.UP:
         case KeyCode.DOWN: {
           let offset = 0;
           if (which === KeyCode.UP) {
             offset = -1;
           } else if (which === KeyCode.DOWN) {
             offset = 1;
+          } else if (isPlatformMac() && ctrlKey) {
```
Comment (afc163) [inline src/OptionList.tsx:199]: How about https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/metaKey

Comment (wendellhu95) [reply src/OptionList.tsx:199]: Command + N would open a new Window, and this cannot be prevented.

**Pull request merged**